Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/isolate-assertions.dfy.expect

```diff
@@ -1,8 +1,8 @@
 Verified 0/2 symbols. Waiting for Foo to verify.
-Verified part 1/3 of Foo, on line 5 (redacted, resource count: 8.7E+002)
-Verified part 2/3 of Foo, on line 6 (redacted, resource count: 3.1E+003)
-Verified part 3/3 of Foo, on line 7 (redacted, resource count: 2.8E+003)
+Verification part 1/3 of Foo, on line 5, verified successfully, redacted and consuming 8.7E+002 resources
```

Comment:
What does "redacted" mean?

Reply:
Because the time taken is not consistent over runs, I replace it with "redacted" in tests using `sed`.

Reply:
I know you replace it with "redacted", my question was really about the meaning of "redacted".
I don't know the rationale behind choosing "redacted" for that replacement. For me, "redacted" means "edit (text) for publication.".
If it was me, I would have just removed the variable part, so that the error would look like "verified successfully and consuming ...."
So my question is, why was the word "redacted" chosen here?

Reply:
I'm using it as a means to say "there was something else here, but it has been removed"